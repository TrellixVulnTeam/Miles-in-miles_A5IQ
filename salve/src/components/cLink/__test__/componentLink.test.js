import React from 'react'
import renderer from 'react-test-renderer';
import ComponentLink from '../componentLink'

describe('Teste de link', () =>{
  it('Should Component Link', () =>{
    const container = (<ComponentLink link="http://www.google.com.br" descricao="Shoppe"/>);
    const tree = renderer
      .create(container)
      .toJSON();
      expect(tree).toMatchSnapshot()
  })
})