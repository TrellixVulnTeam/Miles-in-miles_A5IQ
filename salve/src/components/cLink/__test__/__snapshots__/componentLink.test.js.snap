// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teste de link Should Component Link 1`] = `
<a
  href="http://www.google.com.br"
/>
`;
